app/lib/features/chat/widgets/mention_profile_builder.dart

```diff
@@ -54,55 +54,52 @@ class MentionProfileBuilder extends ConsumerWidget {
           }
           return map;
         });
-        return SizedBox(
-          height: MediaQuery.of(context).size.height * 0.3,
-          child: Card(
-            margin: const EdgeInsets.symmetric(horizontal: 8),
-            elevation: 2,
-            clipBehavior: Clip.hardEdge,
-            shape: RoundedRectangleBorder(
-              borderRadius: BorderRadius.circular(8),
-            ),
-            child: ListView.builder(
-              shrinkWrap: true,
-              padding: const EdgeInsets.all(0),
-              itemCount: users.length,
-              itemBuilder: (_, index) {
-                final id = users.keys.elementAt(index);
-                final displayName = users.values.elementAt(index);
-                return ListTile(
-                  dense: true,
-                  onTap: () {
-                    final autocomplete = MultiTriggerAutocomplete.of(context);
-                    ref
-                        .read(chatInputProvider.notifier)
-                        .addMention(displayName, id);
-                    return autocomplete.acceptAutocompleteOption(
-                      displayName.isNotEmpty ? displayName : id.substring(1),
+        return Card(
+          margin: const EdgeInsets.symmetric(horizontal: 8),
+          elevation: 2,
```

Comment:
what's the reasoning for removing the sized box?

Reply:
The ```SizedBox``` was acting as constraint to limit the height of suggestions list or it was thought to be. But it came to discovery that it is ineffective of resizing layout of ```Overlay``` (Portal) of internal mentions plugin which seems to have its mechanism of sizing using ```widthFactor``` and ```heightFactor```. In cda65d4, you can see I've extended plugin to include ```heightFactor``` param to adjust suggestion height.